addons/portal/static/src/xml/portal_address_warnings.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+    <t t-name="portal.portal_warning_confirmation">
+        <div>
+            <t t-foreach="warnings" t-as="warning">
+                <p>
+                    <t t-foreach="warning" t-as="warning_line">
```

Comment:
you can use t expression directly inside html brols <> (`<p t-foreach="warning" t-as="warning_line">`)
But anyway, I still don't see the point of having multiple warning.
Not sure it's necessary to make a template for this. a confirm dialog isn't inouf ?